interpreter/Makefile

```
@@ -113,7 +113,7 @@ $(WINMAKE):	clean
 # Executing test suite
 
 TESTDIR =	../test/core
-TESTFILES =	$(shell cd $(TESTDIR); ls *.wast)
+TESTFILES =	$(shell cd $(TESTDIR); ls *.wast; ls simd/*.wast)
```

```suggestion
TESTFILES =	$(shell cd $(TESTDIR); ls *.wast; ls */*.wast)
```
(so that it works generically for potential future subdirs)